autoload.php

```diff
@@ -46,7 +46,11 @@
     if (!empty($configFile)) {
         $neonConfig = Neon::decode(file_get_contents($configFile));
         if(is_array($neonConfig) && isset($neonConfig['parameters']) && isset($neonConfig['parameters']['tmpDir'])) {
-            $tmpDir = $neonConfig['parameters']['tmpDir'];
+            // If the tmpDir is configured as an absolute path, use it directly
+            // otherwise, treat the tmpDir as a path relative to the directory of the config file
+            $tmpDir = strpos($neonConfig['parameters']['tmpDir'], '/') === 0
+                ? $neonConfig['parameters']['tmpDir']
+                : dirname($configFile) . '/' . $neonConfig['parameters']['tmpDir'];
```

Comment:
I feel we might not need this bock and rather add the same !isset() check as PHPStan uses: https://github.com/phpstan/phpstan-src/blob/master/src/Command/CommandHelper.php#L228-L231

Reply:
Is it this check that we should replace with the simpler isset: https://github.com/bitExpert/phpstan-magento/pull/163/files/137b8dfd770612cc9cc057d6aaf537cfb650dffc#diff-f79e1f84a3df304b5be8b1bd2f1a36d4bb8417d91e8ae7144e648379d697a57eL48?

If not, I think I need some clarification regarding the change you'd like to see.